omod/src/main/webapp/pages/taggedObjects.gsp

```diff
@@ -29,4 +37,9 @@
         </tr>
    </tbody>
     </table>
+    <div><br/>
+        <button class="cancel" ng-click="navigate()">
+            ${ ui.message("uicommons.return") }
```

Comment:
I think this shouldn't be displayed as a cancel button, just keep it as a regular button